### PR TITLE
Moved KMS code examples into KMS directory; added new KMS code examples

### PR DESCRIPTION
**Pull request by Doug-AWS**
*Issue #, if available:*
KMS had only one code example, in the kms-helloworld directory.

*Description of changes:*
Moved the helloword code into the kms/src/bin directory; added another set of KMS code examples.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws/sdk/examples/kms/src/bin/create-key.rs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+        SubscriberBuilder::default()
+            .with_env_filter("info")
+            .with_span_events(FmtSpan::CLOSE)
+            .init();
+    }
+
+    let conf = Config::builder().region(region).build();
+    let conn = aws_hyper::conn::Standard::https();
+    let client = Client::from_conf_conn(conf, conn);
```
Comment (jdisanti) [inline aws/sdk/examples/kms/src/bin/create-key.rs:57]: I'm thinking we should have most of examples instantiate clients without customizing the connector, like this:
```
    let config = Config::builder().region(region).build();
    let client = Client::from_conf(config);
```

That should show people how to build a client quickly and easily without getting overloaded with details. At the same time, we should have an example somewhere with some good documentation on how to customize the HTTP connection, and maybe some concrete examples of why you would want to do that.

What do you think?

Comment (Doug-AWS) [reply aws/sdk/examples/kms/src/bin/create-key.rs:57]: I agree. I'll make the necessary changes and clear the "waiting-on-author" bit when I'm done.

Comment (Doug-AWS) [reply aws/sdk/examples/kms/src/bin/create-key.rs]: Done.

**Review by rcoh — APPROVED** [review]
LGTM!

**Pull request merged**